osgp/protocol-adapter-iec60870/parent-pa-iec60870/pom.xml

```diff
@@ -0,0 +1,453 @@
+      <url>${maven.site.distributionManagement.site.url}</url>
+    </site>
+  </distributionManagement>
+
+  <properties>
+    <module.baseUri>${project.baseUri}/../</module.baseUri>
+    <osgp.version>${project.version}</osgp.version>
+    <osgp.shared.version>4.35.0-SNAPSHOT</osgp.shared.version>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <spring.version>4.3.18.RELEASE</spring.version>
+    <spring.ws.version>2.4.0.RELEASE</spring.ws.version>
+    <spring.data.version>1.11.6.RELEASE</spring.data.version>
+    <spring.data.key.version>1.2.16.RELEASE</spring.data.key.version>
+    <servlet.version>3.0.1</servlet.version>
+    <hibernate.version>4.2.16.Final</hibernate.version>
+    <hibernate.validator.version>4.3.1.Final</hibernate.validator.version>
+    <junit.version>4.11</junit.version>
+    <flyway.version>4.1.2</flyway.version>
+    <slf4j.version>1.7.7</slf4j.version>
+    <apache.commons.lang.version>3.3.2</apache.commons.lang.version>
+    <apache.httpcomponents.version>4.3.6</apache.httpcomponents.version>
+    <orika.version>1.5.1</orika.version>
+    <netty.version>3.9.4.Final</netty.version> 
+    <maven.project.info.reports.plugin.version>3.0.0</maven.project.info.reports.plugin.version>
+    <maven.site.plugin>3.7.1</maven.site.plugin>
+    <pmd.version>4.3</pmd.version>
+    <apache.activemq.version>5.15.7</apache.activemq.version>
+    <commons.pool.version>1.6</commons.pool.version>
+    <proton-jms.version>0.7</proton-jms.version>
+    <logback.version>1.1.2</logback.version>
+    <logback.ext.version>0.1.2</logback.ext.version>
+    <apache.commons.schema>2.0.3</apache.commons.schema>
+    <maven.compiler.plugin.version>3.2</maven.compiler.plugin.version>
+    <guava.version>18.0</guava.version>
+    <license.maven.plugin>2.11</license.maven.plugin>
+    <hikaricp.version>2.4.0</hikaricp.version>
+    <openmuc.j60870.version>1.2.1</openmuc.j60870.version>
+  </properties>
+
+  <repositories>
+    <repository>
+      <id>fuse-repo</id>
+      <name>Fuse Source</name>
+      <url>http://repo.fusesource.com/nexus/content/groups/public/</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>true</enabled>
+      </snapshots>
+    </repository>
+  </repositories>
+
+  <dependencyManagement>
+    <dependencies>
+      <!-- OSGP -->
+      <dependency>
+        <groupId>org.opensmartgridplatform</groupId>
+        <artifactId>osgp-dto</artifactId>
+        <version>${osgp.shared.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.opensmartgridplatform</groupId>
+        <artifactId>shared</artifactId>
+        <version>${osgp.shared.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>org.opensmartgridplatform</groupId>
+        <artifactId>osgp-iec60870</artifactId>
```

Comment:
This module is not present in this PR? Remove dependency?

Reply:
Removed.